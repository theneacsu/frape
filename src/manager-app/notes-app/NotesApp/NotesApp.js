import React from 'react'
import NotesList from '../components/NotesList/NotesList'
import NotesSearchArea from '../components/NotesSearchArea/NotesSearchArea'
import styles from './NotesApp.module.css'

const NotesApp = props => (
  <div className={styles.wrapperDiv}>
    <div className="container">
      <div className={styles.div}>
        <button
          className={styles.button}
          onClick={() => props.history.push('/notes/create')}
        >
          New Note
        </button>

        <NotesSearchArea />
      </div>
      <NotesList />
    </div>
  </div>
)

export default NotesApp
