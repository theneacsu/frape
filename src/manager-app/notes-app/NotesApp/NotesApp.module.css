.wrapperDiv {
  padding: 2rem;
  background-color: rgb(250, 250, 190);
  flex: 1;
}

.button {
  border: none;
  border-radius: 1rem;
  padding: 1.5rem;
  background-color: #aa76f8;
  color: white;
  font-weight: bold;
  width: 100%;
  margin-bottom: 1.5rem;
  flex: 1;
}

.div {
  margin: 2rem auto;
  width: 100%;
  display: flex;
  flex-direction: column;
}

@media (min-width: 450px) {
  .div {
    flex-direction: row;
    justify-content: space-between;
  }

  .button {
    width: 50%;
  }
}
