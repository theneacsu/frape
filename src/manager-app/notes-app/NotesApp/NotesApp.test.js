import React from 'react'
import { shallow } from 'enzyme'
import NotesApp from './NotesApp'

test('Should render the NotesApp component correctly', () => {
  const wrapper = shallow(<NotesApp />)
  expect(wrapper).toMatchSnapshot()
})
