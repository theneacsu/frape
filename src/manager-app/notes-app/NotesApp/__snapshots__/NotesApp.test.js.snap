// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render the NotesApp component correctly 1`] = `
<div
  className="wrapperDiv"
>
  <div
    className="container"
  >
    <div
      className="div"
    >
      <button
        className="button"
        onClick={[Function]}
      >
        New Note
      </button>
      <Connect(NotesSearchArea) />
    </div>
    <Connect(NotesList) />
  </div>
</div>
`;
